### DISCUSSION TITLE
3D printable OVOS case is now available

### DISCUSSION THREAD

**Discussion by dcveteran** [OpenVoiceOS/ovos-buildroot#221] [Show and tell]
I have spent some time designing an OVOS 3D printable case for use with the Pi4 and 7" Pi touchscreen. This is my first iteration, it has enough space and cutouts to work with different hardware. I may make some tweaks to it later when I have time.

You can see it here. All you need is access to a 3D printer to make your own: 
https://www.printables.com/model/779284-mycroft-or-ovos-raspberry-pi-case

Enjoy!

**Comment by goldyfruit** [comment]
Nice!

**Reply by dcveteran** [reply]
I chipped the top when I was trying to make the opening bigger for the speaker cable. Other than that, this one came out pretty nice.

**Comment by dcveteran** [comment]
I updated this print design to make the parts better fit together.

**Comment by JarbasAl** [comment]
I have my first 3d printer on the way, i'll give this one a shot! also have a 7'inch touchscreen not currently in use
